.github/workflows/snyk-test.yml

```diff
@@ -15,101 +15,45 @@ jobs:
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+      
+  
   snyk_nightly_run:  
       name: Snyk Nightly Run (for nightly cron with JIRA)
       runs-on: ubuntu-latest
       if: github.event_name == 'schedule'
-     
-    
       steps:
-      - name: Check out repository
-        uses: actions/checkout@v2
-
-      - name: Set up Node.js
-        uses: actions/setup-node@v2
-        with:
-          node-version: 18
-
-      - name: Install dependencies
-        run: npm install --legacy-peer-deps
-        working-directory: .github/workflows/
-
-      - name: Install nodejs dependencies
-        run: npm install request-promise --legacy-peer-deps
-        working-directory: .github/workflows/
-
-      - name: Install additional nodejs dependencies
-        run: npm install request --save --legacy-peer-deps
-        working-directory: .github/workflows/
-
-      - name: Install axios
-        run:  npm install axios --save --legacy-peer-deps
-        working-directory: .github/workflows/
-
-      - name: Install jq
-        run: sudo apt-get install -y jq
-        working-directory: .github/workflows/
-
-      - name: Install Snyk CLI
-        run: npm install -g snyk
-        working-directory: .github/workflows/
-
-      - name: Run Snyk test
-        run:  snyk test --all-projects --json > snyk_output.txt || true
-        working-directory: ./
-        env:
-          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
-
-
-      - name: Set environment variables
-        run: |
-          echo "PROJECT_NAME=MDCT" >> $GITHUB_ENV
-          echo "JIRA_PROJECT_KEY=MDCT" >> $GITHUB_ENV
-          echo "JIRA_ISSUE_TYPE=Task" >> $GITHUB_ENV
-          echo "JIRA_LABELS=MCR,snyk" >> $GITHUB_ENV
-          echo "JIRA_EPIC_KEY=MDCT-2280" >> $GITHUB_ENV
-          echo "JIRA_TITLE_PREFIX=[MCR] - Snyk : " >> $GITHUB_ENV
+        - name: Check out repository
+          uses: actions/checkout@v2
+  
+        - name: Install Snyk and Run Snyk test
+          run: |
+            npm install -g snyk
+            snyk test --all-projects --json > snyk_output.txt || true
+          env:
+            SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
           
-      
-      - name: Install Jira library for Node.js
-        run: npm install jira-client  --legacy-peer-deps
-        working-directory: .github/workflows/
-      
-      - name: Run script to parse Snyk output
-        run: node .github/workflows/parse_snyk_output.cjs snyk_output.txt
-        env:
-          JIRA_BASE_URL: qmacbis.atlassian.net
-          JIRA_API_TOKEN: ${{ secrets.JIRA_SERVICE_USER_TOKEN }}
-          JIRA_USER_EMAIL: ${{ secrets.JIRA_SERVICE_USERNAME }}
+        - name: use the custom github  action to parse Snyk output
+          uses: Enterprise-CMCS/macfc-security-scan-report@v1.0.2
+          with:
+              jira-username: ${{ secrets.JIRA_SERVICE_USERNAME }}
+              jira-token: ${{ secrets.JIRA_SERVICE_USER_TOKEN }}
+              jira-host: 'qmacbis.atlassian.net'
```

Comment:
Is it possible to add the MCR application tag here?

`jira-custom-fields: '{ "customfield_14154" : [{"id": "16958", "value": "MCR"}] }'`

Reply:
the tag has been added